addons/crm/static/src/views/forecast_kanban/forecast_kanban_model.js

```diff
@@ -98,7 +99,18 @@ export class ForecastKanbanDynamicGroupList extends ForecastKanbanModel.DynamicG
         this.domain = previousDomain;
         return result;
     }
+
+    /**
+     * @override
+     *
+     * Restore standard behavior for moving records.
+     */
+    async moveRecord(dataRecordId, dataGroupId, refId, targetGroupId) {
```

Comment:
Do we need to override this method?

I know the base override is a bit "useless" in this case, as it will only apply additional behavior if grouped by stage (which should not be the case for the forecast kanban).
But as it does no harm, I would keep it simple and remove this.

As a general rule, fix code should be as short as possible.

Reply:
Hello, 
Good point, I removed it as it's not necessary